src/guacenc/video.h

```diff
@@ -42,16 +50,23 @@
 typedef struct guacenc_video {
 
     /**
-     * Output file stream.
+     * AVStream for video output.
+     * Persists via the AVFormatContext.
```

Comment:
I'm not sure what you mean here by "Persists via the AVFormatContext." Can this comment be clarified?

Reply:
Reworded to be more accurate and clear.